c-boyle/future-hero
# Move Ruffians And Old Lady

## Changes:
- Move ruffians to different table closer to boombox.
- Make the boombox quieter from far away.
- Move old lady to new table.
- Move patrons that were dining near boombox to where the old lady was.
- Add scared particle effect to old lady in ruffian future.

## Comments:
- The animation for the cup falling seems to have been messed up by moving the cup. It still plays in the spot it used to be.

- Some changes will likely also have to be made to match these new positions up with the positions of some characters in https://github.com/c-boyle/future-hero/pull/40

## Screenshots:
Ruffians and old lady in present:
![image](https://user-images.githubusercontent.com/62191940/199375326-f819940b-c06c-4fac-a976-50ce5c139916.png)

Old lady and ruffians in ruffian future:
![image](https://user-images.githubusercontent.com/62191940/199375393-739c4c16-bd4d-4c10-b5d5-188ecc6e16f2.png)

Old lady in puddle future:
![image](https://user-images.githubusercontent.com/62191940/199375518-73de7020-d269-42e8-9eb6-bff6d0732423.png)



### How to Test:
Open and play level, check that every future looks right with the new positions.
